### PR TITLE
[#154514300] Security info

### PR DESCRIPTION
**Pull request by chrisfarms**
# What

* DRYs up the subnav into a partial
* Adds `/security` page documenting security responsibility
* Adds `/ia` page documenting Information Assurance

# How to review

Use `make dev` locally to render the pages and check them out.

# Who can review

Not @chrisfarms

### CODE REVIEW

File: views/ia.erb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+			<p>The platform has been assured by the Cabinet Office senior information risk owner (SIRO) and our information and security risk is deemed to be appropriate for data classified as ‘official’. In order to operate with live services, we underwent a formal risk assessment using a methodology based on ISO27005:2011. Our service is currently subject to the Cabinet Office and the Government Digital Service security governance.</p>
+			<p>Contact us at <a href="mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk">gov-uk-paas-support@digital.cabinet-office.gov.uk</a> if you need details on our Information Risk Management, or <a href="/security">read more about our platform security measures</a>.</p>
+
+			<h2 class="heading-medium">Your service assurance</h2>
+			<p>You’ll be responsible for the assurance of your own service, and making sure that you have in place appropriate controls for the information you handle.</p>
+			<p>When you assure your service through your own organisation’s information assurance (security) process, you don’t need to include assurance of GOV.K PaaS, since we’ve already done that - we can share the work we’ve done with you.</p>
```
Comment (46bit) [inline views/ia.erb]: Typo in GOV.UK PaaS :-)

File: views/ia.erb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+		<div class="column-one-third">
+			<%== erb :'partials/_subnav', :locals => {active: 'ia'} %>
+		</div>
+
+
+		<div class="column-two-thirds">
+			<h1 class="heading-xlarge">Information assurance</h1>
+
+			<h2 class="heading-medium">Platform assurance</h2>
+			<p>The platform has been assured by the Cabinet Office senior information risk owner (SIRO) and our information and security risk is deemed to be appropriate for data classified as ‘official’. In order to operate with live services, we underwent a formal risk assessment using a methodology based on ISO27005:2011. Our service is currently subject to the Cabinet Office and the Government Digital Service security governance.</p>
```
Comment (46bit) [inline views/ia.erb]: Senior Information Risk Owner is capitalised on line 27, and I think should be here given how we immediately acronym it.

File: views/security.erb [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+					<tr>
+						<td>Updating custom buildpacks (if you use them)</td>
+						<td>Updating system buildpacks</td>
+					</tr>
+					<tr>
+						<td>Applying security updates to your Docker images (if you use them)</td>
+						<td>Applying security updates to backing services</td>
+					</tr>
+					<tr>
+						<td>Applying security updates to your Docker images (if you use them)</td>
```
Comment (46bit) [inline views/security.erb]: Duplicate of line 77 :-(

File: views/security.erb [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+						<td>Applying security updates to backing services</td>
```
Comment (46bit) [inline views/security.erb]: Duplicate of line 78

**Review by 46bit — CHANGES_REQUESTED** [review]

**Comment by chrisfarms** [comment]
@46bit 's comments have been addressed.

**Pull request merged**